### DISCUSSION TITLE
Error: Command "npm run build" exited with 1

### DISCUSSION THREAD

**Discussion by Karthik-V26** [vercel/next.js#28237] [Help]
getting this error while deploying to vercel

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
14:03:11.280
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

**Comment by reginald54EE** [comment]
Hey, I had a similar problem. These links should help you solve yours. basically, you have to get rid of all your warnings or set the environment CI variable to false so it treats warnings as warnings instead of errors.
https://stackoverflow.com/questions/66840942/vercel-deployment-error-command-npm-run-build-exited-with-1 
https://github.com/vercel/vercel/discussions/5566
